robot_base_frame: "base_link"
robot_effector_frame: "link_6"
tracking_base_frame: "camera_link"
tracking_marker_frame:  "camera_marker"
calibration_path: "handeye_calibration_result.yaml"
eye_in_hand: true